ddtrace/contrib/internal/cassandra/patch.py

```diff
@@ -0,0 +1,5 @@
+from ddtrace.contrib.internal.cassandra.session import patch
+from ddtrace.contrib.internal.cassandra.session import unpatch
+
+
+__all__ = ["patch", "unpatch"]
```

Comment:
Do we need this file? If cassandra doesn't have a patch module, I don't think we should add it.